### PR TITLE
fix(Logo): handle data load on ssr

### PR DESCRIPTION
**Pull request by tewshi**
on server side rendering, javascript doesn't trigger, so the @error callback is not called, added handler to try or handle cases where the image fails to load (mostly when the file is not found), so that the fallback image can be rendered.

### CODE REVIEW

File: src/components/logos/Logo.vue [hunk 1]
```diff
@@ -1,49 +1,129 @@
 <script setup lang="ts">
-import logo from './logo.svg';
+import fallback from './logo.svg';
+
+interface ExternalLinks {
+  drawer?: string;
+  app?: string;
+  website?: string;
+  about?: string;
+  empty_screen?: string;
```
Comment (kelsos) [inline src/components/logos/Logo.vue:9]: Do you think we can do camelCase as a parameter and use `snakeCase` from `scule` when reading from json?

**Review by kelsos — COMMENTED** [review]
So far this comment and also if we can make some test for the ssr part of the code with vitest

File: src/components/logos/Logo.vue (hunk 1 above)
Comment (tewshi) [reply src/components/logos/Logo.vue:9]: okay, sure thing.

**Review by kelsos — APPROVED** [review]
lgtm, thanks @tewshi

**Pull request merged**